mrc/control/acting/strategies.py

```diff
@@ -0,0 +1,48 @@
+from mrc.control.acting.abstract import AbstractStrategy
+from mrc.control.movement_prediction.target_position_calculator import TargetPositionCalculator
+from mrc.control.steering.exceptions import ObstacleOnTheWayException, SteeringException
+from mrc.shared.position import PolarPosition
+
+
+class FollowMasterStrategy(AbstractStrategy):
+
+    def __init__(self, steering_interface, locator, configurator):
+        self.steering_interface = steering_interface
+        self.locator = locator
+        self.configurator = configurator
+        self.position_calculator = TargetPositionCalculator(self.configurator)
+        self.locations = None
+        self.current_step = (0, 0)
+        self.step_reached = True
+        self.driving = False
+
+    def read(self):
+        self.locations = self.locator.get_locations(None)
+        if self.step_reached:
+            master_position = self.locator.get_locations(self.configurator.master_unit)
+            self.current_step = self.position_calculator.calculate_actual_target_position(master_position,
+                                                                                          self.current_step)
+
+    def think(self):
+        if PolarPosition.are_positions_approximately_same(self.current_step, (0, 0)):
```

Comment:
Since our driving interface should be blocking, update `self.current_step` after `drive_to_point(...)` and remove `self.driving`